hwdb.d/60-keyboard.hwdb

```diff
@@ -231,6 +231,11 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnPredatorPHN16-71:*
  KEYBOARD_KEY_f5=prog1                                  # "predator sense" button
  KEYBOARD_KEY_66=micmute                                # Microphone mute button
 
+# Nitro AN515-58
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnNitro*AN*515-58:*
```

Comment:
`pnNitro*AN*515-58:*` -> `pnNitro*AN*515-58:pvr*`
Otherwise, `AN` and `515-58` can be in other field.